### ISSUE TITLE
OpenSSL on Mac does not build

### ISSUE THREAD

**Issue by Barthelemy** [alisw/alidist#1249]
If one removes openssl from brew on Mac, he/she gets : 
```DEBUG:flpproto:OpenSSL:0: Configured for darwin-i386-cc.
DEBUG:flpproto:OpenSSL:0: 
DEBUG:flpproto:OpenSSL:0: *** Because of configuration changes, you MUST do the following before
DEBUG:flpproto:OpenSSL:0: *** building:
DEBUG:flpproto:OpenSSL:0: 
DEBUG:flpproto:OpenSSL:0: 	make depend
DEBUG:flpproto:OpenSSL:0: + make depend
DEBUG:flpproto:OpenSSL:0: making depend in crypto...
DEBUG:flpproto:OpenSSL:0: clang: error: argument unused during compilation: '-Wa,--noexecstack' [-Werror,-Wunused-command-line-argument]
DEBUG:flpproto:OpenSSL:0: make[1]: *** [local_depend] Error 1
DEBUG:flpproto:OpenSSL:0: make: *** [depend] Error 1
DEBUG:flpproto:OpenSSL:0: 
ERROR:flpproto:OpenSSL:0: Error while executing /Users/aldaqci/alice-jenkins-recipes/sw/SPECS/osx_x86-64/OpenSSL/v1.0.2o-1/build.sh on ```

**Comment by Barthelemy** [comment]
Hi Dario, 
Good that it is in the instructions. It happened to me because I did not update brew before pulling alidist and got the error (wrong version, but same result as not having it at all). I tried to remove it and saw this error and thought that the recipe was broken. 
If the recipe should never be executed on Mac, is there a way to tell it in the recipe itself and to inform the users in a more meaningful way ?
